src/SFA.DAS.EmployerCommitmentsV2/Services/Approvals/Responses/GetSelectDirectTransferConnectionResponse.cs

```diff
@@ -12,4 +12,6 @@ public class TransferConnection
     public string FundingEmployerHashedAccountId { get; set; }
     public string FundingEmployerPublicHashedAccountId { get; set; }
     public string FundingEmployerAccountName { get; set; }
+    public short? Status { get; set; }
```

Comment:
No, but the response could potentially return different status because it's a query parameter, and therefore we have a associated variable called StatusAssignedOn. I would have preferred to call it 'ApprovedOn'